lambdas/services/mock_pds_service.py

```diff
@@ -17,9 +17,8 @@ def pds_request(self, nhs_number: str, *args, **kwargs) -> Response:
                 mock_pds_results.append(json.load(f))
             with open("services/mock_data/pds_patient_9000000002_H81109_gp.json") as f:
                 mock_pds_results.append(json.load(f))
-            with open("services/mock_data/pds_patient_9000000003_H85686_gp.json") as f:
-                mock_pds_results.append(json.load(f))
-            with open("services/mock_data/pds_patient_9000000025_restricted.json") as f:
+
+            with open("services/mock_data/pds_patient_restricted.json") as f:
```

Comment:
Is there a reason for this change? It is removing the stubbed data for one of the ODS codes and replacing the existing restricted patient data with old code

Reply:
Wasn't intentional, might have been something that came through in a merge and got missed. I've reverted this now.

Reply:
Might have come in a merge and been missed. I've changed it back and all the tests pass.